### ISSUE TITLE
How to get Pylint to display code smells in coc.nvim ? (Refactor issues)

### ISSUE THREAD

**Issue by zakissimo** [fannheyward/coc-pyright#637]
As suggested in the documentation I edited my coc-settings.json to display :

```
{
"python.linting.pylintEnabled": true
}
```

Despite that, whenever I write a "smelly" code, I don't get any notifications. (I do get errors if the code doesn't work, but nothing related to [refactoring](https://code.visualstudio.com/docs/python/linting#_pylint)).

Exemple :

```
if foo > 1 and foo < 2:
    return bar
```
This is not flagged, when I should get : "[R1716 chained-comparison] : ["Simplify chained comparison between the operands"] was reported by Pylint."

The valid way being :

```
if 1 < foo < 2:
    return bar
```

**Comment by fannheyward** [comment]
Do you have `python.linting.enabled` enabled?

**Comment by zakissimo** [comment]
I added the line just to be sure, but it says in the documentation that it's enabled by default. In any case enabling it myself didn't change anything.

**Comment by fannheyward** [comment]
Checkout`:CocCommand workspace.showOutput` and select `coc-pyright-linting`, are there any error logs?

**Comment by zakissimo** [comment]
Running the command on this file :
```
#!/usr/bin/env python

def kek(foo, bar):
    if foo > 1 and foo < 2:
        return bar
```
Gives me this output :
```
Using python from /usr/bin/python

########## active linter: pylint
########## Run linter pylint:
{"execPath":"pylint","moduleName":"pylint","args":["--disable=all","--enable=F,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,unused-wildcard-import,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode,E0001,E0011,E0012,E0100,E0101,E0102,E0103,E0104,E0105,E0107,E0108,E0110,E0111,E0112,E0113,E0114,E0115,E0116,E0117,E0118,E0202,E0203,E0211,E0213,E0236,E0237,E0238,E0239,E0240,E0241,E0301,E0302,E0303,E0401,E0402,E0601,E0602,E0603,E0604,E0611,E0632,E0633,E0701,E0702,E0703,E0704,E0710,E0711,E0712,E1003,E1101,E1102,E1111,E1120,E1121,E1123,E1124,E1125,E1126,E1127,E1128,E1129,E1130,E1131,E1132,E1133,E1134,E1135,E1136,E1137,E1138,E1139,E1200,E1201,E1205,E1206,E1300,E1301,E1302,E1303,E1304,E1305,E1306,E1310,E1700,E1701","--msg-template='{line},{column},{category},{msg_id}:{msg} ({symbol})'","--reports=n","--output-format=text","/home/zak/MEGA/apps/ape.py"],"product":1}

########## Linting Output - pylint##########

------------------------------------
Your code has been rated at 10.00/10


Using python from /usr/bin/python

########## active linter: pylint
########## Run linter pylint:
{"execPath":"pylint","moduleName":"pylint","args":["--disable=all","--enable=F,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,unused-wildcard-import,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode,E0001,E0011,E0012,E0100,E0101,E0102,E0103,E0104,E0105,E0107,E0108,E0110,E0111,E0112,E0113,E0114,E0115,E0116,E0117,E0118,E0202,E0203,E0211,E0213,E0236,E0237,E0238,E0239,E0240,E0241,E0301,E0302,E0303,E0401,E0402,E0601,E0602,E0603,E0604,E0611,E0632,E0633,E0701,E0702,E0703,E0704,E0710,E0711,E0712,E1003,E1101,E1102,E1111,E1120,E1121,E1123,E1124,E1125,E1126,E1127,E1128,E1129,E1130,E1131,E1132,E1133,E1134,E1135,E1136,E1137,E1138,E1139,E1200,E1201,E1205,E1206,E1300,E1301,E1302,E1303,E1304,E1305,E1306,E1310,E1700,E1701","--msg-template='{line},{column},{category},{msg_id}:{msg} ({symbol})'","--reports=n","--output-format=text","/home/zak/MEGA/apps/ape.py"],"product":1}

########## Linting Output - pylint##########

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```
